docs/auto-qc/iqms_intepretability.md

```diff
@@ -0,0 +1,1288 @@
+
+
+```python
+# imports
+import os
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import seaborn as sns
+import plotly.express as px
+
+from sklearn.manifold import TSNE
+from sklearn.preprocessing import StandardScaler
+from sklearn.decomposition import PCA
+from sklearn.svm import LinearSVC
+from sklearn.model_selection import GridSearchCV
+from sklearn.model_selection import StratifiedKFold
+from sklearn.linear_model import LogisticRegression, LogisticRegressionCV
+from sklearn.metrics import accuracy_score
+from sklearn.model_selection import train_test_split
+from plotly.subplots import make_subplots
+from sklearn.svm import SVC
+from sklearn.metrics import confusion_matrix, ConfusionMatrixDisplay
+```
+
+First, let's import the data. The [Movement-Related Artifacts (MR-ART)](https://openneuro.org/datasets/ds004173) dataset contains the T1-weighted images of 148 healthy subjects (Nárai et al. 2022). Each subject has been acquired under three motion conditions:
+
+1. no head movement
+2. little head movement
+3. much head movement
+
+The motion was artifically induced by giving the subjects cues when to node their head.
+
+The images were rated by to two expert neuroradiologists, who rated the images in their quality, with ratings
+
+1. good quality
+2. medium quality
+3. bad quality.
+
+The neuroradiologist were asked to harmonize their ratings, hence the dataset contains only one score per images.
+
+
+```python
+# set path to the data
+path_data = os.path.abspath("../data/")
+# import IQMs
+iqms = pd.read_csv(os.path.join(path_data, "IQMs.tsv"), index_col="bids_name", sep="\t")
+# import rating scores
+scores = pd.read_csv(os.path.join(path_data, "scores.tsv"), index_col="bids_name", sep="\t")
+# make sure they are in the same order
+iqms.sort_index(inplace=True)
+scores.sort_index(inplace=True)
+```
+
+Let's combine the motion condition and the manual ratings in one dataframe:
+
+
+```python
+def add_condition_column(scores):
+    """Adds a column with the different movement conditions to the rating scores dataframe"""
+    stand = scores.index.str.contains("standard")
+    hm1 = scores.index.str.contains("motion1")
+    hm2 = scores.index.str.contains("motion2")
+    conditions = [
+        (stand == True),
+        (hm1 == True),
+        (hm2 == True)]
+    choices = [1, 2, 3]
+    scores['condition'] = np.select(conditions, choices)
+    return scores
+scores = add_condition_column(scores)
+```
+
+## Rating scores and image conditions
+
+We can explore how well the raters align in their scores with the motion condition. Does their rating reflect how much people moved in the scanner? Let's plot the confusion matrix of the scores assigned by the rater and the motion condition :
```

Comment:
Line breaks, and please do not use French semicolons (i.e., remove the space between the last word and punctuation)
```suggestion
We can explore how well the raters align in their scores with the motion condition.
Does their rating reflect how much people moved in the scanner?
Let's plot the confusion matrix of the scores assigned by the rater and the motion condition:
```